unlock-protocol-com/blog/email-airdrop-nft.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Unlock Protocol Email Airdropped NFTs
+subTitle: Unlock Protocol added a new feature to email a user when airdropping them an NFT from the creator dashboard.
+authorName: Angela Steffens
+publishDate: August 2, 2022
+description: Unlock Protocol added a new feature to email a user when airdropping them an NFT from the creator dashboard.
+image: /images/blog/email-airdrop-nft/airdrop-nft-share-img.png
```

Comment:
this url does not match the path to the share image. path should be:
/images/blog/email-airdrop-nft/airdrop-header.png